### DISCUSSION TITLE
How to stop/cancel a background job ?

### DISCUSSION THREAD

**Discussion by AiroPi** [matiasdelellis/facerecognition#718] [Q&A] [answered]
I just installed face recognition within community container from nextcloud AIO.
Face recognition started automatically, and now I'm unable to stop it. I want to assign 8GB of RAM instead of 4, but when I do `php occ face:setup -M 8G -m 5`, I only get :
`Another command ('face:background_job') is already running that prevents it from continuing.`

How can I stop it ?

**Comment by matiasdelellis** [comment, marked as answer]
> How can I stop it ?

mm,. It is designed to close properly if you kill it. So just "killall occ".. or maybe search with htop
